CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# `Phoebus` Development Model
+
+----
+For the purpose of our development model, we label _Contributors_ or _Maintainers_ of `Phoebus`. 
+Below we describe these labels and the process for contributing to `Phoebus`.
+
+## Becoming a Contributor
+We welcome contributions from scientists from a large variety of relativistic astrophysics. 
+New users are welcome to contributions to `Phoebus` via the *Contributors* process. 
+Contributors with 6 merged pull requests into the `main` branch (over a minimum of 6 months) 
+will be eligible to join as a *Maintainer* of `Phoebus` with additional repository access and roles. 
+However, final approval of *Maintainer* status will require a unanimous approval by vote by existing *Maintainers*, 
+a necessary step to ensure the safety and integrity of the code base for all users of `Phoebus`.
+
+The _Maintainers_ of `Phoebus` consist of the original developers of the code and those that 
+have a demonstrated history in the development of `Phoebus` prior to the implementation of 
+the _Development Model_ described here. Maintainers hold admin access, serve consistently as 
+reviewers for pull requests, and are in charge of the maintaining, deployment, and improvement of efforts 
+towards: regression testing, documentation, science test cases (gold standards), and continuous integration. 
+
+To maintain their active status as Maintainer, all members will agree to adhere to our 
+Community Code of Conduct and sign a Memorandum of Understanding described [here](GOVERNANCE.md).
+
+# Contributing to `Phoebus`
+
+***Contributors***: (process for most users of `Phoebus`)
+---
+
+1. Create a new fork of `main` where your changes can be made.
```

Comment:
Oops sorry one more thing I just noticed. We can do this later, but we should describe the tests process. As well as the github MR checklist. And the style formatting, since we use clang-format. I *think* we can just pull/adapt all of that from parthenon or singularity docs, once we're on sphinx/RTD, so it can maybe be done in a second sweep.